packages/node_modules/@node-red/editor-client/src/js/nodes.js

```diff
@@ -2419,6 +2419,9 @@ RED.nodes = (function() {
                     if (otherNode.z === n.z) {
                         // Both ends in the same flow/subflow
                         return true
+                    } else if (n.type === "link call") {
+                        // Link call node can call out of a subflow
```

Comment:
Based on previous versions, I should add:

```suggestion
                    } else if (n.type === "link call" && !getSubflow(n.z)) {
                        // Link call node can call out of a subflow but not another one
```

Reply:
Not quite right - this proposed extra check will still allow a link call node that is not in a subflow to call an link inside a subflow.

We need to ensure `otherNode` is not in a subflow.

```suggestion
                    } else if (n.type === "link call" && !getSubflow(otherNode.z)) {
                        // Link call node can call out of a subflow
```